packages/lb-annotation/src/core/toolOperation/pointOperation.ts

```diff
@@ -314,12 +323,10 @@ class PointOperation extends BasicToolOperation {
      */
     if ((this.hoverID === this.selectedID || (this.isMultiMoveMode && this.hoverID)) && e.button === 0) {
       this.dragStatus = EDragStatus.Start;
-      if (this.isMultiMoveMode) {
```

Comment:
why delete it ?

Reply:
I have to get the origin point before i start dragging, which is needed by event 'updatePointByDrag'

Reply:
But i think Keeping this code also can emit `updatePointByDrag`?